Snowflake.Data/Logger/EasyLoggerManager.cs

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+using System;
+using System.IO;
+using log4net;
+using log4net.Appender;
+using log4net.Layout;
+using Snowflake.Data.Configuration;
+
+namespace Snowflake.Data.Log
+{
+    internal class EasyLoggerManager
+    {
+        private static EasyLoggerManager s_instance = null;
+
+        private static readonly object s_instanceLock = new object();
+        private static readonly object s_lock = new object();
+
+        public static EasyLoggerManager Instance
+        {
+            get
+            {
+                lock (s_instanceLock)
```

Comment:
can we return `s_instance` before lock when it is already set?

Reply:
I got rid of locks.